### ISSUE TITLE
Time display irregularity

### ISSUE THREAD

**Issue by thwaller** [baumgarr/nixnote2#330]
I have time set to HH:mm. In the "Note List", notes that are listed as "today" appear in format HH:mm a, or possibly hh:mm a. I cannot say for sure on the leading zero, but it is listing the am/pm, when the settings say it should not.Older notes are listed as specified in preferences.

I am using nixnote2-daily+20170329 on Ubuntu 17.04 64 bit. Ubuntu has its time set as HH:mm as well, so it does not appear that it is pulling a setting from the OS.

**Comment by baumgarr** [comment]
I'm not sure I understand what. Can you please submit a screenshot of the problem and let me know what setting you have in preferences for the time format?

**Comment by baumgarr** [comment]
No response. Closing issue.

**Comment by thwaller** [comment]
My apologies, I had discontinued use of this software. THe time was set to HH:MM. However on the display, only for "today" items, it would display in h:mm AM/PM. The problem still existed at the time that I stopped using. I will be unable to respond with further detail, but I wanted to add a response in case someone else should want to report the problem, so there is documentation that others have seen the problem.
